slices/fontconfig-config.yaml

```diff
@@ -6,7 +6,18 @@ essential:
 slices:
   config:
     essential:
+      - fonts-croscore_fonts
       - fonts-dejavu-core_config
+      - fonts-dejavu-core_fonts
```

Comment:
Is there a reason why `config` should depend on `fonts`? I think these two are decoupled. 

It feels like `config` should be standalone, and there you can have a `core` slice that brings `config` + all `fonts`.